week04_approx_rl/seminar_pytorch.ipynb

```diff
@@ -354,26 +352,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# show video\n",
+    "# Show video. This may not work in some environments. If it doesn't\n",
```

Comment:
"environments" doesn't mean "gym environments" here, does it? Somewhat confusing.